dump.rdb

```diff
@@ -0,0 +1 @@
+REDIS0006�ܳC�Z��V
```

Comment:
This is the redis memory dump, never commit this. Remove it, add it to the .gitignore and untrack it please!